pkg/apis/kubeone/v1beta3/conversion.go

```diff
@@ -30,3 +30,11 @@ func Convert_kubeone_KubeOneCluster_To_v1beta3_KubeOneCluster(in *kubeoneapi.Kub
 func Convert_v1beta3_ContainerRuntimeConfig_To_kubeone_ContainerRuntimeConfig(in *ContainerRuntimeConfig, out *kubeoneapi.ContainerRuntimeConfig, scope conversion.Scope) error {
 	return autoConvert_v1beta3_ContainerRuntimeConfig_To_kubeone_ContainerRuntimeConfig(in, out, scope)
 }
+
+// Convert_v1beta3_CiliumSpec_To_kubeone_CiliumSpec is an autogenerated conversion function.
+func Convert_v1beta3_CiliumSpec_To_kubeone_CiliumSpec(in *CiliumSpec, out *kubeoneapi.CiliumSpec, _ conversion.Scope) error {
```

Comment:
What about `kubeone_CiliumSpec` to `v1beta3_CiliumSpec`? We need it as well because auto-generation for `KubeProxyReplacement` has been disabled in the internal API.